src/disease.c

```diff
@@ -311,7 +326,7 @@ void transition_one_disese_event(
 void transition_to_symptomatic( model *model, individual *indiv )
 {
 	if( gsl_ran_bernoulli( rng, model->params->hospitalised_fraction[ indiv->age_group ] ) )
-		transition_one_disese_event( model, indiv, SYMPTOMATIC, HOSPITALISED, SYMPTOMATIC_HOSPITALISED );
+        transition_one_disese_event( model, indiv, SYMPTOMATIC, HOSPITALISED, SYMPTOMATIC_HOSPITALISED );
```

Comment:
indentation inconsistent with the else statement

Reply:
Fixed indentation.